yaml/roads.yaml

```diff
@@ -353,6 +353,36 @@ filters:
       <<: *osm_footway_properties
       kind: path
       kind_detail: pedestrian
+  - filter:
+      highway: [track]
```

Comment:
You don't need the `[]` around a single value.

Reply:
Fixed in 11d447c59f029103f1a6fdc40125e4771b05519e.